NOTICE

```diff
@@ -2,17 +2,17 @@
 Ohai Notices
 ============
 
-Developed at Opscode (http://www.opscode.com).
+Developed at Chef Software Inc. (http://www.chef.io).
```

Comment:
I had stayed out of contributing since it was just copied from chef/chef at one point and it needs other updates.  I'll do that now.